tests/lua/type_mismatch.lua

```diff
@@ -153,6 +161,10 @@ end
 
 -- Put it to function to limit the lifetime of objects
 generate_tests()
```

Comment:
Can you inline this function?

Reply:
No, we have to restrict the lifetime of variables